src/helpers/token.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Generates an alphanumeric string token of length 10 or 11.
+ */
+export function generateToken() {
+  return Math.random()
```

Comment:
I think this should be fine since this is just for generating an obscure uri, but it's worth being aware that this random generator is not cryptographically secure.

Reply:
Yeah, I'm doing this knowing that it's not cryptographically secure, but I don't think we need a cryptographically secure URI here?

Reply:
don't think so either